worldedit-core/src/main/java/com/sk89q/worldedit/extent/OutputExtent.java

```diff
@@ -94,7 +94,11 @@ default boolean fullySupports3DBiomes() {
      */
     @Deprecated
     default boolean setBiome(BlockVector2 position, BiomeType biome) {
-        return setBiome(position.toBlockVector3(), biome);
+        boolean result = false;
+        for (int y = 0; y < 256; y ++) {
+            result |= setBiome(position.toBlockVector3().mutY(y), biome);
```

Comment:
Not ideal but I can't think of a better solution right now and it works.

Reply:
This is really the only solution; biomes are only stored in 4x4x4 sections, however, doing every 4 blocks could and would cause issues with offsets/transforms, etc